org.eclipse.draw2d.doc.isv/META-INF/MANIFEST.MF

```diff
@@ -2,7 +2,7 @@ Manifest-Version: 1.0
 Bundle-ManifestVersion: 2
 Bundle-Name: %Plugin.name
 Bundle-SymbolicName: org.eclipse.draw2d.doc.isv; singleton:=true
-Bundle-Version: 3.13.300.qualifier
+Bundle-Version: 3.13.400.qualifier
```

Comment:
I assume this increment (same for the other isv plugins) is needed due to slight changes in the generated HTML document caused by the updated AsciiDoctor plugin?